### ISSUE TITLE
Multiple assignment of types is not recognized as a type alias

### ISSUE THREAD

**Issue by dgelessus** [python/mypy#4794]
When assigning a type to more than one name using multiple assignment, none of the new names are recognized as types, and cannot be used in annotations.

I'm using Python 3.6.4 and the current mypy version from the repo (`0.580-dev-80a9b8f8e362902a93606ea0aabf9c18184fef3e`) with no special command-line flags.

Example code:

```python
# Not working:
Spam1 = Ham1 = int
spam1: Spam1  # error: Invalid type "testing2.Spam1"
ham1: Ham1  # error: Invalid type "testing2.Ham1"

# Equivalent and working:
Ham2 = int
Spam2 = Ham2
spam2: Spam2
ham2: Ham2
```

**Comment by gvanrossum** [comment]
I don't think we should enable this. Instead, we should clarify that PEP
484 only endorses type aliases with a single assignment target.

**Comment by ilevkivskyi** [comment]
OK, I am fine with this. @dgelessus would you like to make a PR to python/peps repo?

**Comment by dgelessus** [comment]
Just to clarify, the reason why I ran into this was because I was [writing the stub for `ctypes.wintypes`](https://github.com/python/typeshed/pull/1986/commits/33f9821fdd4a40f19e448469073571d5cdfc3972). Many Windows types have alternate names (mostly historical), which are all mirrored in `ctypes.wintypes`. Because of this, `wintypes.py` has a number of lines like `LPBOOL = PBOOL = _Pointer[BOOL]` or `tagPOINT = _POINTL = POINTL = POINT`, which I copied over into the stub file unmodified. These can of course be split into multiple single assignment statements, and it wouldn't require an unreasonable number of extra lines, but it wouldn't exactly help readability since the original type name needs to be repeated.

I wouldn't have any issue with this if it was just a style guideline - then in the case of `ctypes.wintypes` I could argue ["foolish consistency"](https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds) and ignore it here. However if multiple assignment for type aliases is explicitly prohibited by PEP 484 and mypy, my only option is to split up all the assignments, regardless of whether it actually helps readability.

(To be clear - is the argument for disallowing type aliases by multiple assignment just stylistic, or is there another reason why it's better not to allow it? I don't know mypy's internals, but it doesn't seem like it would be very difficult to implement.)
